### DISCUSSION TITLE
Temporary failure in name resolution

### DISCUSSION THREAD

**Discussion by ozgurerdogan** [psy0rz/zfs_autobackup#193] [General]
Sometime I am getting:
! [Target] STDERR > ssh: Could not resolve hostname backup.servername.com: Temporary failure in name resolution
! [Source] STDERR > mbuffer: error: outputThread: error writing to <stdout> at offset 0x10000: Broken pipe
! [Source] STDERR > mbuffer: warning: error during output to <stdout>: Broken pipe
! [Source] Command "zfs send --large-block --embed --props -i @2023-05-21-10:00:46 D8/vm-239-disk-0@2023-05-21-11:00:48 | mbuffer -q -s128k -m512MB | zstdmt -3 | mbuffer -q -s128k -m16M -R6M" returned exit code 1 (valid codes: [0])

There should not be a dns issue..?

**Comment by psy0rz** [comment]
the error is in the output: _backup.servername.com: Temporary failure in name resolution_

So indeed a dns issue.

**Reply by psy0rz** [reply]
those other errors are just a result of the first error.

**Comment by ozgurerdogan** [comment]
Yes but it should not be dns issue. Maybe some other network issue. Is it ok if I change hostname to ip for existing backup sets...

**Comment by psy0rz** [comment]
Yes you can change the hostname, no problem.

But  backup.servername.com doesnt sound like a valid name?

The error you're seeing is output from ssh, so ssh cannot resolve that hostname. This has nothing to do with zfs-autobackup from that point on.
